opam

```diff
@@ -22,11 +18,10 @@ remove: [["ocamlfind" "remove" "mirage-block"]]
 depends: [
   "base-bytes"
   "cstruct"
-  "mirage-types-lwt" { >= "3.0.0" }
+  "mirage-types-lwt" {>= "3.0.0"}
   "mirage-runtime"
   "lwt"
   "logs"
-  "ocamlbuild" {build}
```

Comment:
Thanks @hannesm, https://github.com/mirage/mirage-dev/pull/264 now includes this.